bench/dune

```diff
@@ -0,0 +1,20 @@
+(executable
+ (name bench)
+ (public_name bench)
+ (package ocamlformat-bench)
+ (libraries bechamel bechamel-js ocamlformat_lib stdio))
+
+(alias
+ (name bench)
+ (deps ./bench.exe))
```

Comment:
What is this rule for ?

Reply:
Same, it came from mirage/index, but I removed it.